diff -Nur sane-frontends-1.0.5.orig/doc/Makefile.in sane-frontends-1.0.5/doc/Makefile.in
--- sane-frontends-1.0.5.orig/doc/Makefile.in	Tue May 29 00:16:53 2001
+++ sane-frontends-1.0.5/doc/Makefile.in	Mon Jul  2 15:28:42 2001
@@ -1,4 +1,5 @@
 SHELL = /bin/sh
+DESTDIR =
 
 VPATH = @srcdir@
 srcdir = @srcdir@
@@ -50,10 +51,10 @@
 	@echo Generating manpage $@...
 
 install: $(MANPAGES)
-	$(MKDIR) $(mandir)/man1
+	$(MKDIR) $(DESTDIR)$(mandir)/man1
 	@for page in $(SECT1); do \
 	  echo installing $${page} in $(mandir)/man1/$${page}...; \
-	  $(INSTALL_DATA) $${page} $(mandir)/man1/$${page} || exit 1; \
+	  $(INSTALL_DATA) $${page} $(DESTDIR)$(mandir)/man1/$${page} || exit 1; \
 	done
 
 uninstall:
diff -Nur sane-frontends-1.0.5.orig/src/Makefile.in sane-frontends-1.0.5/src/Makefile.in
--- sane-frontends-1.0.5.orig/src/Makefile.in	Tue May 29 00:16:55 2001
+++ sane-frontends-1.0.5/src/Makefile.in	Mon Jul  2 15:29:20 2001
@@ -1,4 +1,5 @@
 SHELL = /bin/sh
+DESTDIR =
 
 VPATH = @srcdir@
 srcdir = @srcdir@
@@ -65,12 +66,12 @@
 	  echo "*** maybe the GTK libraries weren't found?" ; \
 	  exit 1 ; \
 	fi
-	$(MKDIR) $(bindir) $(datadir)
+	$(MKDIR) $(DESTDIR)$(bindir) $(DESTDIR)$(datadir)
 	@for program in $(BINPROGS); do \
 	  $(INSTALL_PROGRAM) $${program} \
-	    $(bindir)/$${program}; \
+	    $(DESTDIR)$(bindir)/$${program}; \
 	done
-	$(INSTALL_DATA) $(srcdir)/sane-style.rc $(datadir)/sane-style.rc
+	$(INSTALL_DATA) $(srcdir)/sane-style.rc $(DESTDIR)$(datadir)/sane-style.rc
 
 uninstall:
 	@for program in $(BINPROGS); do \
