--- sane-frontends-1.0.11/src/xscanimage.c.orig	2003-03-23 12:55:43.000000000 +0100
+++ sane-frontends-1.0.11/src/xscanimage.c	2003-06-19 15:40:07.000000000 +0200
@@ -90,6 +90,9 @@
 static void run (char *name, int nparams, GimpParam * param,
 		 int *nreturn_vals, GimpParam ** return_vals);
 
+#  if GIMP_CHECK_VERSION(1,3,15)
+static
+#endif
 GimpPlugInInfo PLUG_IN_INFO =
 {
   NULL,				/* init_proc */
@@ -2023,10 +2026,14 @@
     old_print_func = g_set_print_handler (null_print_func);
     old_printerr_func = g_set_printerr_handler(null_print_func);
     /* gimp_main () returns 1 if xscanimage wasn't invoked by GIMP */
-# ifdef HAVE_OS2_H
+# if GIMP_CHECK_VERSION(1,3,15)
+    result = gimp_main (&PLUG_IN_INFO, argc, argv);
+# else
+#  ifdef HAVE_OS2_H
     set_gimp_PLUG_IN_INFO(&PLUG_IN_INFO);
-# endif
+#  endif
     result = gimp_main (argc, argv);
+# endif
     g_set_print_handler (old_print_func);
     g_set_printerr_handler (old_printerr_func);
     if (result)
